### ISSUE TITLE
0.4 version doesn't load

### ISSUE THREAD

**Issue by JosefNemec** [cooperate/SteamGridDBMetadata#2]
```
2020-05-06 10:50:25.6516|ERROR:Failed to load plugin: Steam Grid DB Metadata ProviderSystem.NullReferenceException: Object reference not set to an instance of an object.
   at SGDBMetadata.SGDBMetadataSettings..ctor(SGDBMetadata plugin) in U:\App_Dev\SGDBMetadata\SGDBMetadata\SGDBMetadata\SGDBMetadataSettings.cs:line 36
   at SGDBMetadata.SGDBMetadata..ctor(IPlayniteAPI api) in U:\App_Dev\SGDBMetadata\SGDBMetadata\SGDBMetadata\SGDBMetadata.cs:line 32
```

0.3 works just fine. Happens only if there are no existing settings present.

**Comment by cooperate** [comment]
I believe I've addressed this in v0.5.  Cannot replicate.

**Comment by JosefNemec** [comment]
Can confirm this doesn't happen with 0.5 anymore.

BTW since you changed the name of extension in manifest, this will now mean that signature of extension changed and installing new version via pext file will create duplicate extension install. I will rework this in Playnite 8, but just so you'll know if people report this to you, because only one plugin instance can be loaded and it's not guaranteed that it will be newest version.
